mobile/src/locale/locale.ts

```diff
@@ -0,0 +1,32 @@
+/******************************************************************************
+ * Copyright (c) Arcadia, Inc. All rights reserved.
+ ******************************************************************************/
+
+import { NativeModules, Platform } from 'react-native';
+import * as moment from 'moment';
+import 'moment/locale/en-gb';
+
+//----------------------------------------------------------------------------
+export function applyLocale() {
+    const defaultLocale = 'en-gb';
```

Comment:
Why 'en-gb'? Just to have some default value? How will it affect UI?

Reply:
A week starts from Monday in GB :)

Reply:
Ah, I see.